force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -217,15 +279,16 @@ public with sharing class FlowControlLoopsCollections {
      * @return A List of integers from 1 to 'n'.
      */
     public static List<Integer> createIntegerList(Integer n) {
-        // Create a new list of integers 
+        // Create a new list of integers
+        List<Integer> numberList = new List<Integer>(); 
 
-        /*
-        for() {
-            //add integer to the list
+        
+        for(Integer counter = 1; counter <= n; counter++) {
+            numberList.add(counter);
```

Comment:
In every programming language since C, it is idiomatic to declare your for loop integer name to be `i`. It's one of the few times you actually name a variable a single character, but in this case I would say it is reasonable to do because you're using a traditional for loop.